### PR TITLE
Improved Orders

### PR DESCRIPTION
**Pull request by XSlayer300**
## About The Pull Request
Improves the Orders' messages by adding more of the said messages.

## Why It's Good For The Game
Adds more variety instead of seeing the same old order messages we see every time.

## Changelog
:cl:
add: Added more order messages
/:cl:

### CODE REVIEW

File: code/datums/emergency_calls/colonists.dm [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 
 /datum/emergency_call/colonist/print_backstory(mob/living/carbon/human/H)
-	to_chat(H, "<B>You are a colonist!</b>")
+	to_chat(H, "<B>You are a simple colonist hoping to be something else.</b>")
```
Comment (DominikPanic) [inline code/datums/emergency_calls/colonists.dm]: This is from another PR, you'll have to fix it.

Comment (XSlayer300) [reply code/datums/emergency_calls/colonists.dm]: Aw great... Reverted now.

**Comment by spookydonut** [comment]
Currently this has no changes to merge.

**Comment by XSlayer300** [comment]
I dunno what am I doing. But I HOPE it should work by now.

File: code/game/objects/machinery/overwatch.dm [hunk 2]
```diff
@@ -1006,17 +1006,17 @@ GLOBAL_LIST_EMPTY(active_laser_targets)
 	switch(command_aura)
 		if("move")
 			var/image/move = image('icons/mob/talk.dmi', src, icon_state = "order_move")
-			message = pick(";GET MOVING!", ";GO, GO, GO!", ";WE ARE ON THE MOVE!", ";MOVE IT!", ";DOUBLE TIME!")
+			message = pick(";GET MOVING!", ";GO, GO, GO!", ";WE ARE ON THE MOVE!", ";MOVE IT!", ";DOUBLE TIME!", ";ONNWARDS!", ";MOVE MOVE MOVE!", ";ON YOUR FEET!", ";GET A MOVE ON!", ";ON THE DOUBLE!", ";ROLL OUT!", ";LET'S GO, LET'S GO!", ";MOVE OUT!", ";LEAD THE WAY!", ";FORWARD!", ";COME ON, MOVE!", ";HURRY, GO!")
```
Comment (factoryman942) [inline code/game/objects/machinery/overwatch.dm]: one N in onwards

**Comment by Rohesie** [comment]
@grinkgo , loremaster?
This looks like it simplifies the stories a lot. Not sure about this.

File: code/game/objects/machinery/overwatch.dm (hunk 2 above)
Comment (XSlayer300) [reply code/game/objects/machinery/overwatch.dm]: Typo fixed.

**Comment by spookydonut** [comment]
Compiles, travis is just being travis

**Pull request merged**